dagshub/data_engine/model/datasource.py

```diff
@@ -1409,9 +1487,18 @@ def get_metadata_entries(self):
         return self._metadata_entries
 
 
+def _get_local_timezone():
+    now_utc = datetime.datetime.utcnow()
+    local_timezone = tz.tzlocal()
+    return now_utc.astimezone(local_timezone).strftime('%z')[:-2] + ':' + now_utc.astimezone(
+        local_timezone).strftime('%z')[-2:]
```

Comment:
https://stackoverflow.com/a/61049837

shouldn't be using utcnow(), because it returns tz-unaware datetimes.
Might also break on windows, where the clock is set to localtime and not UTC.

Can you explain a bit more what this function does in general, because I'm not sure I understand it right now and it feels like there will be some tz errors inevitably

Reply:
return a timezone offset in the form of "+03:00" or "-03:00"